From b183fd1cca0b44a92f0264823dd9f22d2fd8b8d0 Mon Sep 17 00:00:00 2001
From: whyisjake <whyisjake@git.wordpress.org>
Date: Mon, 14 Oct 2019 15:40:04 +0000
Subject: [PATCH] Administration: Ensure that admin referer nonce is valid.

Coding standards, ensure that nonce is valid with identical, rather then equal operator.

Props vortfu, xknown, whyisjake.

Built from https://develop.svn.wordpress.org/trunk@46477


git-svn-id: http://core.svn.wordpress.org/trunk@46275 1a063a9b-81f0-0310-95a4-ce76da25c4cd
---
 wp-includes/pluggable.php | 4 ++--
 wp-includes/version.php   | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/wp-includes/pluggable.php b/wp-includes/pluggable.php
index 206761aaef95..fe6822d21056 100644
--- a/wp-includes/pluggable.php
+++ b/wp-includes/pluggable.php
@@ -1106,7 +1106,7 @@ function auth_redirect() {
 	 *                   0-12 hours ago, 2 if the nonce is valid and generated between 12-24 hours ago.
 	 */
 	function check_admin_referer( $action = -1, $query_arg = '_wpnonce' ) {
-		if ( -1 == $action ) {
+		if ( -1 === $action ) {
 			_doing_it_wrong( __FUNCTION__, __( 'You should specify a nonce action to be verified by using the first parameter.' ), '3.2.0' );
 		}
 
@@ -1125,7 +1125,7 @@ function check_admin_referer( $action = -1, $query_arg = '_wpnonce' ) {
 		 */
 		do_action( 'check_admin_referer', $action, $result );
 
-		if ( ! $result && ! ( -1 == $action && strpos( $referer, $adminurl ) === 0 ) ) {
+		if ( ! $result && ! ( -1 === $action && strpos( $referer, $adminurl ) === 0 ) ) {
 			wp_nonce_ays( $action );
 			die();
 		}
diff --git a/wp-includes/version.php b/wp-includes/version.php
index e53607b467a4..76be1c0883c2 100644
--- a/wp-includes/version.php
+++ b/wp-includes/version.php
@@ -13,7 +13,7 @@
  *
  * @global string $wp_version
  */
-$wp_version = '5.3-beta3-46476';
+$wp_version = '5.3-beta3-46477';
 
 /**
  * Holds the WordPress DB revision, increments when changes are made to the WordPress DB schema.
